src/repart/repart.c

```diff
@@ -310,9 +310,9 @@ static void copy_files_free_many(CopyFiles *f, size_t n) {
 }
 
 typedef enum SubvolumeFlags {
```

Comment:
Let's just move this enum to src/basic/btrfs.h and rename it to `BtrfsSubvolFlags` please instead of duplicating it.